qiskit/circuit/library/standard_gates/i.py

```diff
@@ -48,6 +48,11 @@ def __init__(self, label: Optional[str] = None):
         """Create new Identity gate."""
         super().__init__("id", 1, [], label=label)
 
+    def _define(self) -> None:
+        from qiskit.circuit import QuantumCircuit  # pylint: disable=cyclic-import
+
+        self.definition = QuantumCircuit(1, name=self.name)
```

Comment:
```suggestion
        self.definition = QuantumCircuit(1, name=self.name)
        self.definition.u(0, 0, 0, 0)
```

Reply:
I don't think this achieves anything; if a backend wants to assign further meaning to the operation then they can put it in their `basis_gates` in which case this definition will never get used, and if it _isn't_ in `basis_gates`, then unitary optimisation will remove the gate with-or-without this extra gate in the definition.